scripts/tokenizers/get_hf_text_data.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This script is used to download text corpus from HuggingFace datasets,
+where the saved corpus can be further used to train a tokenizer using `process_asr_text_tokenizer.py`.
+
+Usage:
+```
+python get_hf_text_data.py --config-path="conf" --config-name="huggingface_data_tokenizer"
+```
+
+Please refer to "conf/huggingface_data_tokenizer.yaml" for more details.
+"""
+
+
+import os
+from itertools import repeat
+from multiprocessing import Pool
+from pathlib import Path
+
+import datasets as hf_datasets
+from omegaconf import OmegaConf, open_dict
+
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+
+
+def clean_text(text: str, symbols_to_keep=None):
```

Comment:
By default, do we want to normalize the text by default ? Or keep it as an option disabled by default ?

Reply:
hmmm, let's disable it by default